client/signup/steps/site-type/form.jsx

```diff
@@ -44,29 +52,88 @@ class SiteTypeForm extends Component {
 		this.props.submitForm( type );
 	};
 
-	renderSegmentOptions() {
-		return getAllSiteTypes().map( siteTypeProperties => (
-			<Card
-				className="site-type__option"
-				key={ siteTypeProperties.id }
-				displayAsLink
-				data-e2e-title={ siteTypeProperties.slug }
-				tagName="button"
```

Comment:
Question: We seem to be changing the `<button />` to `<div />` - could you please explain the rationale behind this change? If there's no reason for it, It looks like an accessibility regression.

Reply:
Added back.